scripts/validate_data_building.py

```
@@ -0,0 +1,610 @@
+        if group_file.exists():
+            group_df = pd.read_csv(group_file, encoding="utf-8-sig")
+            if "Date" in group_df.columns:
+                sample_date = group_df["Date"].iloc[0]
+                date_formats["group.csv"] = {"format": "DD/MM/YYYY", "sample": sample_date}
+                
+                # Test parsing
+                try:
+                    parsed = pd.to_datetime(sample_date, format="%d/%m/%Y", dayfirst=True)
+                    date_formats["group.csv"]["parse_success"] = True
+                    date_formats["group.csv"]["parsed_sample"] = parsed
+                except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)